src/config/models.py

```diff
@@ -109,18 +109,19 @@ class AdvancedOptionsModel(_IniBaseModel):
     exit_key: str
     log_lvl: str = "info"
     process_name: str = "Diablo IV.exe"
+    import_build: str
```

Comment:
please keep it sorted alphabetically :)

Reply:
Completely missed, these were alphabetical. Cheers!